.github/workflows/release.yml

```diff
@@ -39,27 +34,36 @@ jobs:
 
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: all
```

Comment:
Looking at this project QEMU is not needed. You can do native cross compilation using the global scope ARG `BUILDPLATFORM` as shown in https://github.com/crazy-max/goreleaser-xx#minimal